storage/innobase/include/fil0fil.h

```diff
@@ -1619,13 +1619,47 @@ and a suffix. The string must be freed by caller with ut_free().
 @param[in] path NULL or the directory path or the full path and filename.
 @param[in] name NULL if path is full, or Table/Tablespace name
 @param[in] suffix NULL or the file extention to use.
+@param[in] trim_name true if the last name on the path should be trimmed.
 @return own: file name */
 char*
+fil_make_filepath_low(
+	const char*	path,
+	const char*	name,
+	ib_extention	suffix,
+	bool		trim_name);
+
+/** Wrapper function over fil_make_filepath_low to build file name.
+@param[in] path NULL or the directory path or the full path and filename.
+@param[in] name NULL if path is full, or Table/Tablespace name
+@param[in] suffix NULL or the file extention to use.
+@param[in] trim_name true if the last name on the path should be trimmed.
+@return own: file name */
+static inline char*
 fil_make_filepath(
 	const char*	path,
 	const char*	name,
 	ib_extention	suffix,
-	bool		strip_name);
+	bool		trim_name) {
```

Comment:
Please insert a line break before any `{` that starts the function body, so that Emacs `narrow-to-defun` and similar functions will continue to work. Only for member functions we deviate from this rule.

Also, I think that new functions would better be declared or defined in the common formatting style, with multiple parameters per line, no TAB and no `[in]` after `@param`.

Reply:
Done